src/Neo/Ledger/MemoryPool.cs

```diff
@@ -293,7 +293,7 @@ internal VerifyResult TryAdd(Transaction tx, DataCache snapshot)
         {
             var poolItem = new PoolItem(tx);
 
-            if (_unsortedTransactions.ContainsKey(tx.Hash)) return VerifyResult.AlreadyExists;
+            if (_unsortedTransactions.ContainsKey(tx.Hash)) return VerifyResult.AlreadyInPool;
```

Comment:
We also need to adjust the behaviour of `NeoSystem.ContainsTransaction`, aren't we?

Reply:
https://github.com/neo-project/neo/blob/27df5ce9e45ae224258c7515d9e239a7ceb6d4a7/src/Neo/Ledger/Blockchain.cs#L340-L345
and
https://github.com/neo-project/neo/blob/27df5ce9e45ae224258c7515d9e239a7ceb6d4a7/src/Neo/NeoSystem.cs#L277-L281

Reply:
Good catch!

Reply:
Resolved.